src/arch/LowLevelWindow/LowLevelWindow_SDL.cpp

```diff
@@ -0,0 +1,258 @@
+#include "global.h"
+#include "LowLevelWindow_SDL.h"
+#include "arch/arch_default.h"
+#include "RageLog.h"
+
+#include <SDL3/SDL.h>
+
+#include "DisplaySpec.h"
+#include "Preference.h"
+#include "RageSurface.h"
+#include "RageSurface_Load.h"
+
+Preference<bool> g_bSDLLinuxUseWayland("SDLLinuxUseWayland", true);
```

Comment:
this makes the game crash under X11:
```
Error: There was an error while initializing your video card.

Please do not file this error as a bug!  Use the web page below to troubleshoot this problem.

http://old.stepmania.com/stepmaniawiki.php?title=Video_Driver_Troubleshooting

Video Driver: OpenGL

Initializing opengl...
SetVideoMode failed: Failed to create SDL Window - Check logs;Failed to create SDL Window - Check logs;Failed to create SDL Window - Check logs;No display resolutions
```

I think we should probe what we're running under instead of forcing wayland